source/docs/hardware/hardware-basics/status-lights-ref.rst

```diff
@@ -6,202 +6,529 @@ Many of the components of the FRC Control System have indicator lights that can
+| LEDs Alternate               | Off/Orange     | CAN bus detected, robot disabled           |
++------------------------------+----------------+--------------------------------------------+
+| LEDs Alternate               | Off/Slow Red   | CAN bus/PWM is not detected                |
++------------------------------+----------------+--------------------------------------------+
+| LEDs Alternate               | Off/Fast Red   | Fault Detected                             |
++------------------------------+----------------+--------------------------------------------+
+| LEDs Alternate               | Red/Orange     | Damanged Hardware                          |
++------------------------------+----------------+--------------------------------------------+
+| LEDs Strobe towards (M-)     | Off/Red        | Forward Limit Switch or Forward Soft Limit |
++------------------------------+----------------+--------------------------------------------+
+| LEDs Strobe towards (M+)     | Off/Red        | Reverse Limit Switch or Reverse Soft Limit |
++------------------------------+----------------+--------------------------------------------+
+| LED1 Only (closest to M+/V+) | Green/Orange   | In Boot-loader                             |
++------------------------------+----------------+--------------------------------------------+
+
+Status LEDs During Calibration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
++------------------------+------------------------+
+| Status LEDs Blink Code | Talon SRX State        |
++========================+========================+
+| Flashing Red/Green     | Calibration Mode       |
++------------------------+------------------------+
+| Blinking Green         | Successful Calibration |
++------------------------+------------------------+
+| Blinking Red           | Failed Calibration     |
++------------------------+------------------------+
+
+B/C CAL Blink Codes
+^^^^^^^^^^^^^^^^^^^
+
++----------------------+-----------------+
+| B/C CAL Button Color | Talon SRX State |
++======================+=================+
+| Solid Red            | Brake Mode      |
++----------------------+-----------------+
+| Off                  | Coast Mode      |
++----------------------+-----------------+
 
 Spike relay configured as a motor, light, or solenoid switch
 ------------------------------------------------------------
 
 .. image:: images/status-lights/spikeRelay1Light.png
+  :width: 600
+
++---------------------------------+-------------+-----------+-------------------------------------+
+| Inputs                          | Outputs     | Indicator | Motor Function                      |
++-----------------+---------------+------+------+           |                                     |
+| Forward (White) | Reverse (Red) | M+   | M-   |           |                                     |
++=================+===============+======+======+===========+=====================================+
```

Comment:
I think you need to change this back. On a separate note... What if you tried just bolding the words?

Reply:
Whoops, good catch; thanks. Here's what it would look like with bolded title text.
![image](https://user-images.githubusercontent.com/15172054/80818685-86b45600-8b88-11ea-9e99-e2eb02f34cd8.png)

Reply:
Looks awesome

Reply:
Added in 2213050.